### PR TITLE
Update Accessibility Details

### PR DESCRIPTION
**Pull request by titusfortner**
### Description
* Add image of what the integration looks like
* Add implementation details for Ruby, Python & Java with: 
  * Requirements
  * Usage
  * Example Code 
* Add links to all relevant code bases, projects, and documentation

### Motivation and Context
Per the conversation in this ticket: https://saucedev.atlassian.net/browse/TCD-1668

Focusing solely on wdio is useful to only ~10% of our user base.

The vast majority of the existing content for wdio is a rewrite of the information found at https://webdriver.io/docs/gettingstarted & https://webdriver.io/docs/sauce-service and I think it only complicates the information we are trying to demonstrate by including it in this document.

Also, there is a more complicated example in the wdio code for `getAxeResults()`, but there isn't any example or info on why to do that, or what can be done with it, so I removed it. Unlike the other languages, I don't have an actual code example to point to. Maybe @wswebcreation can give guidance. Finally, we link to https://www.npmjs.com/package/@axe-core/webdriverio but I don't understand how to match the documentation they have there with what we are providing here (different variables ; but maybe that's because I'm not a JS dev.

### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist
- [x] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/master/CONTRIBUTING.MD) document.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

Resolves: [TCD-1668](https://saucedev.atlassian.net/browse/TCD-1668)

### CODE REVIEW

File: docs/basics/integrations/deque.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ keywords:
 
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
+import useBaseUrl from '@docusaurus/useBaseUrl';
 
 <p><small><span class="highlight beta">BETA</span></small></p>
```
Comment (spider-sauce) [inline docs/basics/integrations/deque.md:16]:
```suggestion

```

**Review by wswebcreation — APPROVED** [review]
Hi @titusfortner and @spider-sauce 

Thanks for the PR and making this _better_ and more usable for our customers. I do have 1 question. Do we have more information about what the impact of a poor accessible website is, so we can explain in a few sentences how this can help improving that?
Maybe we have a video or something.

**Comment by titusfortner** [comment]
Just added 3 things:

1. Changed the headers to just be bold text to keep the TOC clean (we could put them back and remove the TOC if that's better)
2. Made it more clear why someone should visit the Deque docs ( @wswebcreation's concern) 
3. SeleniumAxeDotnet just released a version that has been verified to work with our product, so I added more info about it.

File: docs/basics/integrations/deque.md [hunk 2]
```diff
@@ -11,44 +11,91 @@ keywords:
...
+{ label: 'C#', value: 'dotnet', }
+]
+}>
 
-## Set Up Dependencies
+<TabItem value="java">
 
-Add the following dependencies to your `package.json` file:
+We highly recommend using the Java [Sauce Bindings](https://opensource.saucelabs.com/sauce_bindings/).
+Check out the description and examples for the new [Acessibility functionality](https://opensource.saucelabs.com/sauce_bindings/docs/accessibility).
```
Comment (spider-sauce) [inline docs/basics/integrations/deque.md:50]:
```suggestion
Check out the description and examples for the new [Accessibility functionality](https://opensource.saucelabs.com/sauce_bindings/docs/accessibility).
```

File: docs/basics/integrations/deque.md [hunk 3]
```diff
@@ -11,44 +11,91 @@ keywords:
...
-* `axe-core` for Accessibility tests (`a11y` results)
-* `sauce-service` for the WebdriverIO plugin service
+The following is information on how to use the Java Deque Axe Selenium library directly:
 
-```js
-"@axe-core/webdriverio": "4.2.1"
-"@wdio/sauce-service"
+__Requirements__
+
+Add this to your `pom.xml`
```
Comment (spider-sauce) [inline docs/basics/integrations/deque.md:56]:
```suggestion
Add this to your `pom.xml`:
```

File: docs/basics/integrations/deque.md [hunk 4]
```diff
@@ -11,44 +11,91 @@ keywords:
...
+<dependency>
+  <groupId>com.deque.html.axe-core</groupId>
+  <artifactId>selenium</artifactId>
+  <version>4.2.2</version>
+</dependency>
+```
+
+__Usage__
+
+This method will populate results on the Sauce Labs Accessibility tab:
```
Comment (spider-sauce) [inline docs/basics/integrations/deque.md:69]:
```suggestion
This method populates results on the Sauce Labs Accessibility tab:
```

File: docs/basics/integrations/deque.md [hunk 5]
```diff
@@ -11,44 +11,91 @@ keywords:
...
-## Create the Configuration
 
-Next, create a `wdio.conf.js` in your root project directory, see the [WebdriverIO documentation](https://webdriver.io/docs/gettingstarted#set-up) for further details. 
+</TabItem>
+<TabItem value="wdio">
+ 
+__Requirements__
+
+* [Install and Set up](https://webdriver.io/docs/gettingstarted/) WebdriverIO
+  * This documentation also shows you how to run a test locally which will help troubleshoot potential issues with your code before running on Sauce Labs
```
Comment (spider-sauce) [inline docs/basics/integrations/deque.md:87]:
```suggestion
  * This documentation shows you how to run a test locally, and also how to troubleshoot potential issues with your code before running on Sauce Labs
```

File: docs/basics/integrations/deque.md [hunk 6]
```diff
@@ -58,236 +105,130 @@ before: function (capabilities, specs, browser) {
...
     return axeWdio.analyze()
 })
 ```
 
-When the `analyze()` command is called, `axe-core` then scans the current document content.
 
-The final step is to define a local variable to store the violations (critical, moderate, and/or minor) and return the `result`:
+__Usage__
 
+This method will populate results on the Sauce Labs Accessibility tab:
```
Comment (spider-sauce) [inline docs/basics/integrations/deque.md:119]:
```suggestion
This method populates results on the Sauce Labs Accessibility tab:
```

File: docs/basics/integrations/deque.md [hunk 7]
```diff
@@ -58,236 +105,130 @@ before: function (capabilities, specs, browser) {
...
-values={[
-{label: 'Chrome', value: 'chrome'},
-{label: 'Firefox', value: 'firefox'},
-]}>
+</TabItem>
+<TabItem value="python">
 
-<TabItem value="chrome">
+We highly recommend using the Python [Sauce Bindings](https://opensource.saucelabs.com/sauce_bindings/).
+Check out the description and examples for the new [Acessibility functionality](https://opensource.saucelabs.com/sauce_bindings/docs/accessibility).
```
Comment (spider-sauce) [inline docs/basics/integrations/deque.md:129]:
```suggestion
Check out the description and examples for the new [Accessibility functionality](https://opensource.saucelabs.com/sauce_bindings/docs/accessibility).
```

File: docs/basics/integrations/deque.md [hunk 8]
```diff
@@ -58,236 +105,130 @@ before: function (capabilities, specs, browser) {
...
-        'goog:chromeOptions': {
-            args: [
-                '--no-sandbox', 
-                '--disable-infobars',
-            ],
-        },
-    },
-];
-```
+The following is information on how to use the Python sa11y library directly:
```
Comment (spider-sauce) [inline docs/basics/integrations/deque.md:131]:
```suggestion
The following is information on how to use the Python `sa11y` library directly:
```

File: docs/basics/integrations/deque.md [hunk 9]
```diff
@@ -58,236 +105,130 @@ before: function (capabilities, specs, browser) {
...
+from sa11y.analyze import Analyze
+```
 
-### Running the Test
-Now in your actual tests, all you need to do is call the following to get the `axe-core` test results:
+__Usage__
 
-```js
-browser.getAxeResults()
+This method will populate results on the Sauce Labs Accessibility tab:
```
Comment (spider-sauce) [inline docs/basics/integrations/deque.md:147]:
```suggestion
This method populates results on the Sauce Labs Accessibility tab:
```

File: docs/basics/integrations/deque.md [hunk 10]
```diff
@@ -58,236 +105,130 @@ before: function (capabilities, specs, browser) {
...
-```js
-const {config} = require('./wdio.conf');
+</TabItem>
+<TabItem value="ruby">
 
-config.user = "yourUserName";
-config.key = "yourAccessKey";
-config.region = "yourRegion -> us / eu";
+We highly recommend using the Ruby [Sauce Bindings](https://opensource.saucelabs.com/sauce_bindings/). 
+Check out the description and examples for the new [Acessibility functionality](https://opensource.saucelabs.com/sauce_bindings/docs/accessibility).
```
Comment (spider-sauce) [inline docs/basics/integrations/deque.md:162]:
```suggestion
Check out the description and examples for the new [Accessibility functionality](https://opensource.saucelabs.com/sauce_bindings/docs/accessibility).
```

File: docs/basics/integrations/deque.md [hunk 11]
```diff
@@ -58,236 +105,130 @@ before: function (capabilities, specs, browser) {
...
 
-config.capabilities = [
-    
-];
+The following is information on how to use the Ruby sa11y library directly:
```
Comment (spider-sauce) [inline docs/basics/integrations/deque.md:164]:
```suggestion
The following is information on how to use the Ruby `sa11y` library directly:
```

File: docs/basics/integrations/deque.md [hunk 12]
```diff
@@ -58,236 +105,130 @@ before: function (capabilities, specs, browser) {
...
+```ruby
+require 'sa11y/analyze'
 ```
 
-You can retrieve your Sauce Labs account credentials with [this link](https://app.saucelabs.com/user-settings). The `region` property is used for both Virtual Machine Cloud and Real Device Cloud tests, and the acceptable values are:
+__Usage__
 
-* `us` - *default*
-* `eu`
+This method will populate results on the Sauce Labs Accessibility tab:
```
Comment (spider-sauce) [inline docs/basics/integrations/deque.md:180]:
```suggestion
This method populates results on the Sauce Labs Accessibility tab:
```

File: docs/basics/integrations/deque.md [hunk 13]
```diff
@@ -58,236 +105,130 @@ before: function (capabilities, specs, browser) {
...
-<TabItem value="safari">
+__Usage__
 
-```js
-    {
-        browserName: 'safari', 
-        platformName: 'macOS 10.15',
-        browserVersion: 'latest',
-    },
+This method will populate results on the Sauce Labs Accessibility tab:
```
Comment (spider-sauce) [inline docs/basics/integrations/deque.md:213]:
```suggestion
This method populates results on the Sauce Labs Accessibility tab:
```

**Review by spider-sauce — CHANGES_REQUESTED** [review]
A couple of nitpicky passive voice and typos comments. Other than that it LGTM 👍

File: docs/basics/integrations/deque.md (hunk 5 above)
Comment (titusfortner) [reply docs/basics/integrations/deque.md:87]: What about
> This documentation shows you how to run a test locally, which allows you to troubleshoot potential issues with your code before running on Sauce Labs

**Review by spider-sauce — APPROVED** [review]

**Pull request merged**